### PR TITLE
Get Aria label for inputs

### PR DESCRIPTION
**Pull request by kandula-aswin**

### CODE REVIEW

File: wcomponents-theme/src/main/js/wc/dom/getLabelsForElement.js [hunk 1]
```diff
@@ -78,35 +94,36 @@ define(["wc/dom/tag",
...
+						result = getAncestorLabel(element);
+					}
+				}
 			}
-
-			// try getting an ancestor label element ONLY if element is input, textarea, select or progress.
-			tagName = element.tagName;
-			if (~LABELABLE.indexOf(tagName)) {
-				return getAncestorLabel(element);
+			for (var i = 0; i < ariaLabel.length; i++) {
```
[the remark this answers is not in the document]
Comment (marksreeves) [reply wcomponents-theme/src/main/js/wc/dom/getLabelsForElement.js]: @ricksbrown we should add the vars-on-top rule to the eslint config to enforce this.

**Comment by ricksbrown** [comment]
This PR is missing a changelog entry

File: wcomponents-theme/src/main/js/wc/dom/getLabelsForElement.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ricksbrown) [reply wcomponents-theme/src/main/js/wc/dom/getLabelsForElement.js]: @marksreeves Totally agree.

**Comment by kandula-aswin** [comment]
Closing this PR, as `aria-label` branch as it became a mush up between #1079 and #1401. Will create another branch from 'Freddie' and resubmit the enhancement

**Pull request closed without merging**